src/Uno.UI/Helpers/WinUI/StringUtil.cs

```diff
@@ -28,7 +28,7 @@ internal static string FormatString(string format, params object[] parms)
 			// the output string, as the C++ index is staring at 1.
 			list.Insert(0, null);
 
-			return string.Format(CultureInfo.CurrentUICulture, dotnetFormat, list.ToArray());
+			return string.Format(CultureInfo.InvariantCulture, dotnetFormat, list.ToArray());
```

Comment:
I don't know the callers for this method
but that could be problematic if used inside UI (or user visible) code

Reply:
It's used in many locations where culture is relevant: https://github.com/Youssef1313/uno/blob/f28af93f14d38e2f2dcc084a75615ec0bcb4f8b1/src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs#L221

I think this should stay as `CurrentUICulture`.

Reply:
```suggestion
			return string.Format(CultureInfo.CurrentUICulture, dotnetFormat, list.ToArray());
```

Reply:
I guess it will show up as an error in CI, though CA1305 documentation doesn't clearly state why `CurrentUICulture` isn't allowed for `string.Format`

Reply:
`CurrentCulture` would be enough then, they're equivalent as far as Uno is concerned.

Reply:
@jeromelaban This applies to all occurrences where you suggested `CurrentUICulture`?